### PR TITLE
Implements base class of unit test

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: src/test/java/funda/prog101/BolierTest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/BolierTest.java:4]: This line could't exist

File: src/test/java/funda/prog101/CoffeeFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/CoffeeFilterTest.java:4]: This line could't exist

File: src/test/java/funda/prog101/HeaterTest.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/HeaterTest.java:4]: This line could't exist

File: src/test/java/funda/prog101/MarkIVTest.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/MarkIVTest.java:4]: This line could't exist

File: src/test/java/funda/prog101/PotTest.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/PotTest.java:4]: This line could't exist

File: src/test/java/funda/prog101/SensorTest.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package test.java.funda.prog101;
+
+import static org.junit.Assert.*;
+
```
Comment (Enrique-C) [inline src/test/java/funda/prog101/SensorTest.java:4]: This line could't exist

**Review by Enrique-C — COMMENTED** [review]
This commit only has a mistake

File: src/test/java/funda/prog101/SensorTest.java (hunk 6 above)
Comment (JuanCarlosMartinez4) [reply src/test/java/funda/prog101/SensorTest.java:4]: Ok i will Update unit test classes

**Pull request closed without merging**